docs/ref/settings.txt

```diff
@@ -910,6 +910,38 @@ This is an Oracle-specific setting.
 
 The maximum size that the DATAFILE_TMP is allowed to grow to.
 
+.. setting:: DATA_UPLOAD_MAX_MEMORY_SIZE
+
+DATA_UPLOAD_MAX_MEMORY_SIZE
+---------------------------
+
+.. versionadded:: 1.9
+
+Default: ``2621440`` (i.e. 2.5 MB).
+
+The maximum size (in bytes) that a request body may be before a
+:exc:`~django.core.exceptions.SuspiciousOperation` is raised. This value is applied
+when accessing ``request.body`` or ``request.POST`` and is calculated against
+the total request size excluding any file upload data. You can set it to
+``None`` to disable the check. When processing forms with very large field values,
+or with a large number of fields, this limit may need to be increased.
+
+See also :setting:`FILE_UPLOAD_MAX_MEMORY_SIZE`.
+
+.. setting:: DATA_UPLOAD_MAX_NUMBER_FIELDS
+
+DATA_UPLOAD_MAX_NUMBER_FIELDS
+-----------------------------
+
+.. versionadded:: 1.9
+
+Default: ``1000``
+
+The maximum number of parameters received either via GET, or FORM POST before a
```

Comment:
If this applies now applies to GET requests too, I'm not sure the `DATA_UPLOAD` name is still a good one. Maybe drop the "UPLOAD" part or think of something entirely different?